### ISSUE TITLE
px-timeline wrong icons used, icons not centered, arrows instead of arrowheads used, etc.

### ISSUE THREAD

**Issue by Lizrichardson53** [predix-ui/predix-ui.github.io#152]
## **Chrome:**
**All below applies to both dark and light themes**
In enhanced view:
Wrong icons used, wrong size, not centered (binding to arrays & polymer 1.0), arrowhead supposed to be used instead of arrow.
![image](https://user-images.githubusercontent.com/30634949/28843328-9c600264-76b5-11e7-8e63-4a91170af327.png)
![image](https://user-images.githubusercontent.com/30634949/28843338-a5f4a924-76b5-11e7-8af9-30cfa4823ca7.png)

In enhanced view & show node content:
Arrowhead supposed to be used instead of arrow, wrong icons used, icons not centered (Polymer 1.0, too much spacing between quotation mark and quote, Check stroke width and type boldness for headers and subheaders.
![image](https://user-images.githubusercontent.com/30634949/28843371-c8786a12-76b5-11e7-85a3-99eb27f4eb86.png)
![image](https://user-images.githubusercontent.com/30634949/28843377-d0a6e862-76b5-11e7-92e7-538279fb41c0.png)

In enhanced compact editor info:
Arrowheads supposed to be used instead of arrows
![image](https://user-images.githubusercontent.com/30634949/28843299-7c6393d6-76b5-11e7-8642-a617538622ab.png)
![image](https://user-images.githubusercontent.com/30634949/28843306-83637ae8-76b5-11e7-9ae9-ff0dcb31e9de.png)

In time groups:
Vertical timeline line doesn't line up with other line when enhanced is turned on
![image](https://user-images.githubusercontent.com/30634949/28843491-3e60f73a-76b6-11e7-8007-6fcb9ce51782.png)

With show node content, enhanced and compact editor info on:
Wrong spacing between header, percentage and body text
![image](https://user-images.githubusercontent.com/30634949/28843787-4a5719f6-76b7-11e7-8b57-46cf9fbc023e.png)
![image](https://user-images.githubusercontent.com/30634949/28843806-55badc7e-76b7-11e7-949a-41ce81470c7a.png)
![image](https://user-images.githubusercontent.com/30634949/28843816-5b697aea-76b7-11e7-9d36-b35c97900391.png)

**In Dark theme** 
with show node content and enhanced turned on:
Stroke width too heavy for underline of topics
![image](https://user-images.githubusercontent.com/30634949/28843618-b27886d8-76b6-11e7-9b70-16dd203cf1e1.png)
![image](https://user-images.githubusercontent.com/30634949/28843636-c3dc2c7c-76b6-11e7-9ac0-6ef5d20a9fdf.png)

## **Safari 10.3**
Dark & Light Themes, Enhanced on:
Wrong icons used
![image](https://user-images.githubusercontent.com/30634949/28843885-9537fa3a-76b7-11e7-9383-5c137b4c2ec8.png)
![image](https://user-images.githubusercontent.com/30634949/28843896-9c90b678-76b7-11e7-82c4-965d6cb617a4.png)

Dark & Light themes, Enhanced & show node content on:
Wrong icons used
![image](https://user-images.githubusercontent.com/30634949/28843943-e509c7e6-76b7-11e7-9af9-2fdb807808a7.png)
![image](https://user-images.githubusercontent.com/30634949/28843953-f076ef50-76b7-11e7-841f-101324ba9bee.png)
Subheaders not bolded
![image](https://user-images.githubusercontent.com/30634949/28844033-2352829a-76b8-11e7-845d-20865f879a09.png)
![image](https://user-images.githubusercontent.com/30634949/28844042-294974c4-76b8-11e7-9474-58eb4288aa84.png)

Dark & Light themes, enhanced & compact editor on:
Wrong icons used:
![image](https://user-images.githubusercontent.com/30634949/28844157-96a706d0-76b8-11e7-89f6-565687bdee99.png)

Dark & Light themes, once show node content is clicked on with time groups and enhanced on, information and line jumps
![image](https://user-images.githubusercontent.com/30634949/28844256-e0fd1f44-76b8-11e7-868e-ee57c50e74dd.png)
![image](https://user-images.githubusercontent.com/30634949/28844276-ece333e8-76b8-11e7-9470-e50e35b58355.png)

Dark Theme, Enhanced & show node content on:
Stroke width for topics too heavy
![image](https://user-images.githubusercontent.com/30634949/28844091-5717c7ac-76b8-11e7-8ecd-49c790e76440.png)
![image](https://user-images.githubusercontent.com/30634949/28844098-5d9466d0-76b8-11e7-8d95-668738801c32.png)

**Comment by randyaskin** [comment]
I updated a lot of these and told Justice about it right after we started the QA blitz. The chevron has been fixed. The icons are different because I chose new ones that are more optimized for the larger icon size used in timeline - they are all arbitrary and chosen in the data structure of the demo page anyway. The weight of the fonts, bubbles and connecting lines were not explicitly spec'ed and I assumed they appear a little thicker because all of the specs that Wade gave me from Sketch have this weird "somewhat bolder" appearance - I'll ask Justice if that was not his intention, but we pretty consistently use 1px lines everywhere in the design system.
